sdcm/cluster.py

```diff
@@ -1413,6 +1421,23 @@ def get_node_database_errors(self):
                 errors.append({node.name: node_errors})
         return errors
 
+    def nodes_in_other_dc(self, node, node_list):
+        ret = []
+        for n in node_list:
+            if n.dc_idx != node.dc_idx:
+                ret.append(n)
+        return ret
```

Comment:
can do it in one line:
return [n for n in node_list if n.dc_idx != node.dc_idx]